.github/workflows/deprecations.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Crawl Deprecations
+
+on:
+  push:
+    branches:
+      - setup-git-action
+
+jobs:
+  crawl-and-update:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: git fetch --no-tags --prune --depth=5 origin master
+      - name: crawl
+        run: npx deprecation-crawler --path ./deprecation-crawler.config.github-action.json --tag master
+      - uses: EndBug/add-and-commit@v4 # You can change this to use a specific version
+        with:
+          add: 'deprecations/*'
```

Comment:
Does this also commit the changes made to the target repo?

Reply:
Yes. for now to a testing branch: https://github.com/timdeschryver/deprecation-manager/commit/9782830f66fe1de289fd849987d19b138a20c8ec